packages/ag-charts-community/src/dom/domManager.ts

```diff
@@ -89,6 +90,10 @@ export class DOMManager extends BaseManager<Events['type'], Events> {
     private readonly observer?: IntersectionObserver;
     private readonly sizeMonitor = new SizeMonitor();
 
+    public readonly seriesWidget: NativeWidget<HTMLElement>;
+    public readonly chartWidget: NativeWidget<HTMLElement>;
+    public readonly containerWidget: NativeWidget<HTMLElement>;
+
```

Comment:
I don't think this is the right spot for these:
- `DOMManager` is mostly concerned with lower-level DOM primitives (e.g. raw `HTMLElement`s)
- use and configuration of these seems to happen in other parts of the codebase:
  -  bounds being set in `SeriesAreaManager`
   - event listeners being setup in `RegionManager`
   - references are pushed in `root` and `series` regions for use there